engine/swarm/admin_guide.md

```diff
@@ -231,29 +247,46 @@ you demote or remove a manager
-of temporary node failures (machine reboots or crash with restart).
+of temporary node failures (machine reboots or crash with restart) or other
+transient errors. However, a swarm cannot automatically recover if it loses a
+quorum.
 
-In a swarm of `N` managers, there must be a quorum of manager nodes greater than
-50% of the total number of managers (or `(N/2)+1`) in order for the swarm to
-process requests and remain available. This means the swarm can tolerate up to
-`(N-1)/2` permanent failures beyond which requests involving swarm management
-cannot be processed. These types of failures include data corruption or hardware
-failures.
+In a swarm of `N` managers, a quorum (a majority) of manager nodes must always
+be available. For example, in a swarm with 5 managers, a minimum of 3 must be
+operational and in communication with each other. In other words, the swarm can
+tolerate up to `(N-1)/2` permanent failures beyond which requests involving
+swarm management cannot be processed. These types of failures include data
+corruption or hardware failures.
+
+### Recovering from losing the quorum
+
+If you lose the quorum of managers, you cannot administer the swarm. If you have
+lost the quorum and you attempt to perform any management operation on the swarm,
+an error occurs:
+
+```no-highlight
+Error response from daemon: rpc error: code = 4 desc = context deadline exceeded
+```
 
-Even if you follow the guidelines here, it is possible that you can lose a
-quorum of manager nodes. If you can't recover the quorum by conventional
-means such as restarting faulty nodes, you can recover the swarm by running
-`docker swarm init --force-new-cluster` on a manager node.
+The best way to recover from losing the quorum is to bring the failed nodes back
+online. If you can't do that, the only way to recover from this state is to use
+the `--force-new-cluster` action from a manager node. This removes all managers
+except the manager the command was run from. The quorum is achieved because
+there is now only one manager. Promote nodes to be managers until you have the
+desired number of managers.
 
 ```bash
 # From the node to recover
 docker swarm init --force-new-cluster --advertise-addr node01:2377
 ```
 
-The `--force-new-cluster` flag puts the Docker Engine into swarm mode as a
-manager node of a single-node swarm. It discards swarm membership information
-that existed before the loss of the quorum but it retains data necessary to the
-Swarm such as services, tasks and the list of worker nodes.
+When you run the `docker swarm init` command with the `--force-new-cluster` flag
+, the Docker Engine where you run the command becomes the manager node of a
+single-node swarm which is capable of managing and running services. The manager
+has all the previous information about services and tasks, but you need to add
+manager and worker nodes to achieve your previous task distribution and ensure
```

Comment:
Workers would still be part of the swarm, but you would need to re-add managers.